starter_code/public/javascript/APIHandler.js

```diff
@@ -1,25 +1,42 @@
 class APIHandler {
-  constructor (baseUrl) {
+  constructor(baseUrl) {
     this.BASE_URL = baseUrl;
   }
 
-  getFullList () {
-
+  getFullList() {
+    return $.get(`${this.BASE_URL}characters`).catch(err => {
+      console.log("Error");
+    });
   }
 
-  getOneRegister () {
-
+  getOneRegister(id) {
+    return $.get(`${this.BASE_URL}characters/` + id)
```

Comment:
si estás utilizando los literales, con las comillas invertidas estas ` ` el + id podrías ponerlo con ${id} igual que has hecho con el this.BASE_URL